### PR TITLE
- added docker support

### PR DESCRIPTION
**Pull request by jan-schutte**

### CODE REVIEW

File: src/SampleWebApp/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Commands.CreateAllNoxTypeCommand.g.cs
Comment (rochar) [inline src/SampleWebApp/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Commands.CreateAllNoxTypeCommand.g.cs:1]: you are removing the generated code, just rebuild the solution...

File: src/SampleWebApp/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+COPY ["Nox.Secrets.Azure/Nox.Secrets.Azure.csproj", "Nox.Secrets.Azure/"]
+COPY ["Nox.Secrets/Nox.Secrets.csproj", "Nox.Secrets/"]
+COPY ["Nox.Secrets.Hashicorp/Nox.Secrets.Hashicorp.csproj", "Nox.Secrets.Hashicorp/"]
+COPY ["Nox.Localization/Nox.Localization.csproj", "Nox.Localization/"]
+COPY ["Nox.Logging.Serilog/Nox.Logging.Serilog.csproj", "Nox.Logging.Serilog/"]
+COPY ["Nox.Monitoring.ElasticApm/Nox.Monitoring.ElasticApm.csproj", "Nox.Monitoring.ElasticApm/"]
+RUN dotnet restore "SampleWebApp/SampleWebApp.csproj"
+COPY . .
+WORKDIR "/src/SampleWebApp"
+RUN dotnet build "SampleWebApp.csproj" -c Release -o /app/build
```
Comment (rochar) [inline src/SampleWebApp/Dockerfile:30]: should it be in debug? the intention its to develop with docker?

**Review by rochar — APPROVED** [review]
looks good, one comment and one question

File: src/SampleWebApp/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Commands.CreateAllNoxTypeCommand.g.cs
Comment (jan-schutte) [reply src/SampleWebApp/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Commands.CreateAllNoxTypeCommand.g.cs:1]: added back

**Pull request merged**